src/Microsoft.Azure.WebJobs.Host/Queues/Bindings/QueueBindingProvider.cs

```diff
@@ -195,22 +181,33 @@ private IAsyncCollector<IStorageQueueMessage> BuildFromQueueAttribute(QueueAttri
             return new QueueAsyncCollector(queue, _messageEnqueuedWatcherGetter.Value);
         }
 
-        private static IStorageQueue GetQueue(QueueAttribute attrResolved)
+        private IStorageQueue GetQueue(QueueAttribute attrResolved)
```

Comment:
That would then require all the callers upstream to be async - I think the intent was to keep this localized.